### ISSUE TITLE
Training Set size

### ISSUE THREAD

**Issue by anushl9o5** [Li-Chengyang/MSDS-RCNN#5]
In the paper, there is a mention of **7,601 training images** being used. But also has indicated following the method used in **Illumination-aware faster r-cnn for robust multispectral pedestrian detection** that uses **25,086 training images**. Is there any further processing or filtering done to reduce them to 7,601 images ?

**Comment by Li-Chengyang** [comment]
Yes, we filtered the training data. You can refer to papers for details of our filtering method.

**Comment by maartenvds** [comment]
What about the labels 'cyclist' or 'people' (group of pedestrians)? If a training image contains one of them, do you include the image for training or not? (not mentioned in the paper)
